src/process_utility.c

```diff
@@ -2659,7 +2696,20 @@ timescaledb_ddl_command_start(
 		return;
 	}
 
-	if (!process_ddl_command_start(&args))
+	/*
+	 * Process Utility/DDL operation locally then pass it for execution
+	 * on any remote servers.
+	 */
+	args.hcache = ts_hypertable_cache_pin();
+
+	handled = process_ddl_command_start(&args);
+
+	if (ts_cm_functions->ddl_command_start)
```

Comment:
So I am a bit skeptical of this hook. It happens /before/ the action itself happens on the frontend (and such actions will sometimes throw errors when running on the frontend). Are we sure we want it running before  `prev_ProcessUtility(&args)` below?